routes/index.js

```diff
@@ -0,0 +1,9 @@
+const router = require('express').Router();
+const usersRoutes = require('./users.routes.js')
+const authRoutes = require('./auth.routes.js');
+const { BearerAuth } = require('../middleware/auth.js');
+
+router.use('/users', BearerAuth, usersRoutes)
+router.use('/auth', authRoutes)
+
+module.exports = router;
```

Comment:
Suppose we have 100s of route files. So, I don't think so it sounds good to create 100s lines of code to represent them. Please apply a better approach to call router. use no matter how many routes files are defined